src/identity/interaction/email-password/handler/EmailPasswordGetResetPasswordHandler.ts

```diff
@@ -0,0 +1,43 @@
+import assert from 'assert';
+import { parse } from 'url';
+import { getLoggerFor } from '../../../../logging/LogUtil';
+import type { HttpHandlerInput } from '../../../../server/HttpHandler';
+import { HttpHandler } from '../../../../server/HttpHandler';
+import type { RenderHandler } from '../../../../server/util/RenderHandler';
+import type { EmailPasswordResetPasswordRenderHandler } from './EmailPasswordResetPasswordRenderHandler';
+
+/**
+ * Handles the creation of the Reset Password form after
+ * the user clicks on it from the link provided in the email.
+ */
+export class EmailPasswordGetResetPasswordHandler extends HttpHandler {
```

Comment:
Would just merge this with the previous one.

Reply:
Personally, I disagree. I think the the handlers for a Get request and a Post request should be separate.